src/sysusers/sysusers.c

```diff
@@ -1235,6 +1310,7 @@ static int add_user(Context *c, Item *i) {
         i->todo_user = true;
         log_info("Creating user '%s' (%s) with UID " UID_FMT " and GID " GID_FMT ".",
                  i->name, strna(i->description), i->uid, i->gid);
+        log_audit_op(c, AUDIT_ADD_USER, "adding user", i->name, i->uid, /* success= */ true);
```

Comment:
hmm, does this make sense? shouldn't we just generate these success messages at the very end too, once we know we managed to put the new passwd file into place?

i.e. it appears to me log_audit_failures() should be log_audit_results() and generate  both failure and success events, and no other call should.

Reply:
The reason I did it this way to avoid the situation where the account is created and we are interrupted before we manage to log about it. But maybe this is too much paranoia. I'll change it to only log on success.